assets/pages/digital-subscription-landing/components/digitalSubscriptionLandingHeader.jsx

```diff
@@ -40,7 +40,7 @@ const gridPicture = {
   fallback: 'digitalSubscriptionHeaderDesktop',
   fallbackSize: 500,
   altText: 'digital subscription',
-  fallbackImageType: 'png',
+  fallbackImgType: 'png',
```

Comment:
It is possible to add the type to this constant, and maybe would be useful to mark props of FridPicture as a exact match https://github.com/guardian/support-frontend/blob/9093558e04028e70990dbf89137c2687873b8df2/assets/components/gridPicture/gridPicture.jsx#L31

I.E. wrappinf the type with  `|` (more info [here](https://flow.org/en/docs/types/objects/#toc-exact-object-types))